src/elastic-search/elastic-search.controller.ts

```diff
@@ -0,0 +1,138 @@
+import {
```

Comment:
We need to implement permissions?
Do we need access to the underlying elastic search from scicat?

Reply:
Yes we login to elastic search from scicat, but I don't think we need to implement permissions. Once elastic search is enabled anyone should be able to search through elastic search. And let the Datasource controller or service layer to decide what data to show which group of people

Reply:
Check how dataset controller implements the authorization.
When I check the code the ES endpoint did not specify any authorization on any of them!!!